src/SFA.DAS.ProviderApprenticeshipsService.Web/Views/Shared/ChooseEmployer.cshtml

```diff
@@ -0,0 +1,71 @@
+﻿@using SFA.DAS.ProviderApprenticeshipsService.Web.Models
+@using SFA.DAS.ProviderApprenticeshipsService.Web.Models.Types
+@model SFA.DAS.ProviderApprenticeshipsService.Web.Models.ChooseEmployerViewModel
+@{
+    switch (Model.EmployerSelectionAction)
```

Comment:
This could maybe be in a mapper, rather than in the razor? Titles, descriptions, url actions, etc?

Reply:
A html mapper/helper?